Source/Core/Core/HW/WiimoteEmu/Camera.cpp

```diff
@@ -53,7 +53,7 @@ int CameraLogic::BusWrite(u8 slave_addr, u8 addr, int count, const u8* data_in)
   return RawWrite(&m_reg_data, addr, count, data_in);
 }
 
-void CameraLogic::Update(const Common::Matrix44& transform)
+void CameraLogic::Update(const Common::Matrix44& transform, Common::Vec2 field_of_view)
```

Comment:
Just curious, is there a reason this isn't `const&`?

Reply:
Not really a reason. It's just two floats so passing by-value is maybe possibly more efficient than referencing them, but I can change it.